narwhal/worker/src/tests/worker_tests.rs

```diff
@@ -1,3 +1,4 @@
+/*
```

Comment:
I'll refactor those as well - just commented out temporarily to speed up opening the PR